include/mata/nfa.hh

```diff
@@ -824,31 +717,13 @@ public:
     const_iterator begin() const { return const_iterator::for_begin(this); }
     const_iterator end() const { return const_iterator::for_end(this); }
 
-    const TransitionList& operator[](State state) const
+    const Moves& operator[](State state) const
     { // {{{
-        assert(state < transitionrelation.size());
+        assert(state < transition_relation.size());
 
-        return transitionrelation[state];
+        return transition_relation[state];
     } // operator[] }}}
 
-    /**
-     * Return all epsilon transitions from epsilon symbol under a given state.
-     * @param[in] state State from which are epsilon transitions checked
-     * @param[in] epsilon User can define his favourite epsilon or used default
-     * @return Returns reference element of transition list with epsilon transitions or end of transition list when
-     * there are no epsilon transitions.
-     */
-    TransitionList::const_iterator get_epsilon_transitions(State state, Symbol epsilon = EPSILON) const;
-
-    /**
-     * Return all epsilon transitions from epsilon symbol under given state transitions.
-     * @param[in] state_transitions State transitions from which are epsilon transitions checked.
-     * @param[in] epsilon User can define his favourite epsilon or used default
-     * @return Returns reference element of transition list with epsilon transitions or end of transition list when
-     * there are no epsilon transitions.
-     */
-    static TransitionList::const_iterator get_epsilon_transitions(const TransitionList& state_transitions, Symbol epsilon = EPSILON) ;
-
```

Comment:
Why are we removing these functions? They are kind of important for when you work with NFAs with epsilons.

Reply:
I removed them because there were not used anywhere and we want to make interface lightweighted.

Reply:
These functions were made in preparation for getting epsilon transitions from alphabets for when we actually use alphabets anywhere, for example, in the next iteration of our tool Noodler which needs to work with epsilon transitions. I do not think these functions should be therefore removed.

Reply:
Ok, I put it back.